rumqttc/examples/async_manual_acks_v5.rs

```diff
@@ -65,7 +65,10 @@ async fn main() -> Result<(), Box<dyn Error>> {
             // Its important not to block notifier as this can cause deadlock.
             let c = client.clone();
             tokio::spawn(async move {
-                c.ack(&publish).await.unwrap();
+                let mut ack = c.get_manual_ack(&publish);
+                ack.set_reason(ManualAckReason::Success);
```

Comment:
Should the reason be success by default and settable if something else is desired? I imagine success is desired for a majority of scenarios.

Reply:
default reason code for puback is [success](https://github.com/bytebeamio/rumqtt/blob/c5bfdd74a535cadd518c12f7667e64f1ec68858c/rumqttc/src/v5/mqttbytes/v5/puback.rs#L30). You don't need to call that `set_reason` unless you wish to set the reason code as failure.